R/data.R

```diff
@@ -38,6 +38,14 @@ NULL
 #' @keywords data
 NULL
 
+#' Seed geographical dictionary in Portuguese
+#'
+#' @name data_dictionary_newsmap_pt
+#' @docType data
+#' @author Barbara Ellynes Zucchi Nobre Silva \email{xx@yyy}
```

Comment:
Which email should write here? It is common to use email address as an person identifier in open source projects.

Reply:
You can use the: barbara@zucchi.science